CryptoAnalysis/src/main/java/crypto/exceptions/CryptoAnalysisException.java

```diff
@@ -0,0 +1,10 @@
+package crypto.exceptions;
+
+public class CryptoAnalysisException extends RuntimeException {
+	
+	private static final long serialVersionUID = 1L;
+
+	public CryptoAnalysisException(String message) {
```

Comment:
I'm sure this also exist in Java: Please create an additional constructor that accepts a message and an exception, which will be the "inner exception" (in java is think this is called "cause")